models/demos/mamba/tt/model_config.py

```diff
@@ -10,7 +10,7 @@
 def create_model_config(batch_size, hidden_size, mode=ModelMode.DECODE, seq_len=1):
     configs = {}
     latent = 32
-    configs["core_grid_row"] = 5
+    configs["core_grid_row"] = 8
```

Comment:
For CI to pass we need to set
WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml

Reply:
It's already present in `tests/scripts/single_card/nightly/run_wh_b0_only.sh,`